src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -108,14 +157,51 @@ public IDisposable On(string methodName, Type[] parameterTypes, Func<object[], o
+        /// </returns>
+        /// <remarks>
+        /// This is a low level method for invoking a streaming hub method on the server. Using an <see cref="HubConnectionExtensions"/> <c>StreamAsChannelAsync</c> extension method is recommended.
+        /// </remarks>
         public async Task<ChannelReader<object>> StreamAsChannelCoreAsync(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default) =>
             await StreamAsChannelCoreAsyncCore(methodName, returnType, args, cancellationToken).ForceAsync();
 
+        /// <summary>
+        /// Invokes a hub method on the server using the specified method name, return type and arguments.
+        /// </summary>
+        /// <param name="methodName">The name of the server method to invoke.</param>
+        /// <param name="returnType">The return type of the server method.</param>
+        /// <param name="args">The arguments used to invoke the server method.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None" />.</param>
+        /// <returns>
+        /// A <see cref="Task{TResult}"/> that represents the asynchronous invoke.
+        /// The <see cref="Task{TResult}.Result"/> property returns an <see cref="object"/> for the hub method return value.
+        /// </returns>
+        /// <remarks>
+        /// This is a low level method for invoking a hub method on the server. Using an <see cref="HubConnectionExtensions"/> <c>InvokeAsync</c> extension method is recommended.
+        /// </remarks>
         public async Task<object> InvokeCoreAsync(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default) =>
             await InvokeCoreAsyncCore(methodName, returnType, args, cancellationToken).ForceAsync();
 
-        // REVIEW: We don't generally use cancellation tokens when writing to a pipe because the asynchrony is only the result of backpressure.
-        // However, this would be the only "invocation" method _without_ a cancellation token... which is odd.
+        /// <summary>
+        /// Invokes a hub method on the server using the specified method name and arguments.
+        /// Does not wait for a response from the receiver.
+        /// </summary>
+        /// <param name="methodName">The name of the server method to invoke.</param>
+        /// <param name="args">The arguments used to invoke the server method.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None" />.</param>
+        /// <returns>A <see cref="Task"/> that represents the asynchronous invoke.</returns>
+        /// <remarks>
+        /// This is a low level method for invoking a hub method on the server. Using an <see cref="HubConnectionExtensions"/> <c>SendAsync</c> extension method is recommended.
```

Comment:
`Using an <see cref="HubConnectionExtensions"/> <c>SendAsync</c> extension method`

This looks a little odd. Maybe just `cref` the `SendAsync` method directly?

Reply:
I did this because you can't reference the group of overloads with that name using with cref, just a single type.

Reply:
Really? I though you could reference a method group... I'm OK with something like this if that's not possible. Maybe `Using the <c>SendAsync</c> extension method on <see cref="HubConnectionExtensions" /> is recommended.`